platform/node/DEVELOPING.md

```diff
@@ -48,13 +48,13 @@ To compile the Node.js bindings and install module dependencies, from the reposi
 #### MacOS
 
 ```bash
-cmake . -B build -G Ninja -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_BUILD_TYPE=Debug -DMBGL_WITH_COVERAGE=ON
+cmake . -B build -G Ninja -DMLN_WITH_NODE -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_BUILD_TYPE=Debug -DMBGL_WITH_COVERAGE=ON
```

Comment:
Shouldn't this be -DMLN_WITH_NODE=ON ? same with line 57

Reply:
I think all should stay `MGL` for now and then we rename all at once. Otherwise it's just confusing.

Reply:
I will keep it, but make a separate PR to rename the other prefixes.

I don't want to introduce new `MBGL_*` options.

Reply:
Can you also pick these PRs to the `opengl-2` branch (before I setup automatic picking)?

Reply:
Sure.